readme.txt

```diff
@@ -475,162 +475,3 @@ For more FAQs, please visit the [FAQ section on GitHub here](https://github.com/
 
 = 1.0 =
 * First Version. Basic map creation and marker creation.
-
-== Upgrade Notice ==
```

Comment:
@hupe13 Why remove this?

Reply:
I checked Leaflet Map with the [Plugin Checker](https://wordpress.org/plugins/plugin-check/). The readme.txt has two warnings: too many tags (max 5) and upgrade_notice_limit: The upgrade notice for "3.1.0" exceeds the limit of 300 characters. 

I investigated further: The Upgrade Notice seems to be ignored from readme.txt. I found [this](https://andidittrich.com/2015/05/howto-upgrade-notice-for-wordpress-plugins.html#magic_action_hook_8), but this is [outdated](https://github.com/WordPress/wordpress-develop/blob/8a52d746e9bb85604f6a309a87d22296ce1c4280/src/wp-admin/includes/update.php#L587-L616). It seems you need to write an upgrade notice like [in this comment](https://developer.wordpress.org/reference/hooks/in_plugin_update_message-file/).